e2e/worker/test.sh

```diff
@@ -22,6 +22,9 @@ add_trap() {
     trap "(set +e; $traps)" EXIT
 }
 
+# fresh start since test.sh is run in sequential steps
+bazel clean
```

Comment:
@thesayyn If I don't bazel clean then the 2nd run of ./test.sh fails with 5.3.2 on test case 11. Might be worth trying to see if there is a potential bug there.